mcs/nunit24/NUnitMocks/mocks/AssemblyInfo.cs

```diff
@@ -6,7 +6,3 @@
 
 using System;
 using System.Reflection;
-
-[assembly: AssemblyDelaySign(false)]

```

Comment:
There is a good argument in favor of what you did, even if signing -- this stuff is too variable to go in sources, belongs in the command line...